test/ngSanitize/sanitizeSpec.js

```diff
@@ -266,14 +268,25 @@ describe('HTML', function() {
     });
   });
 
+  // See https://github.com/cure53/DOMPurify/blob/a992d3a75031cb8bb032e5ea8399ba972bdf9a65/src/purify.js#L439-L449
```

Comment:
Since they are feature sniffing, I was not sure if we needed to mention the type and version of the browser... I expect that Firefox do not consider their part a bug, which means that it will never be resolved. But in any case, there may be other browsers out there that exhibit the same behaviour.

Is the aim of these comments to indicate when we can remove it?